addons/IWB_Toolbar/src/presenter.js

```diff
@@ -2457,6 +2489,129 @@ function AddonIWB_Toolbar_create() {
 
         presenter.stopwatchAdded = true;
     };
+    
+    presenter.getLeftValue = function (popupType) {
+        const panelLeftValue = parseInt(presenter.$panel.css('left'), 10);
+        const toolbarWidth = parseInt(presenter.$panel.outerWidth(), 10);
+
+        return panelLeftValue + toolbarWidth + GAP_VALUE + presenter.getHorizontalOffset(popupType);
+    }
+
+    presenter.getHorizontalLeftValue = function (popupType) {
+        const panelLeftValue = parseInt(presenter.$panel.css('left'), 10);
+        const offset = presenter.getHorizontalPopupOffset(popupType);
+
+        return panelLeftValue + offset;
+    }
+
+    presenter.getHorizontalPopupOffset = function (popupType) {
+        switch (popupType) {
+            case presenter.POPUP_TYPE.STOPWATCH:
+                return 0;
+
+            case presenter.POPUP_TYPE.CLOCK:
+                return presenter.getStopwatchWidth();
+
+            case presenter.POPUP_TYPE.NOTE:
+                return presenter.getStopwatchWidth() + presenter.getClockWidth() + presenter.notesCounter * GAP_VALUE;
```

Comment:
czy presenter.notesCounter nie powinno być zależne od tego, że popup zmienił pozycję ? 
Stwórz kilka notatek a potem przesuń addon. Nowe notatki będą się tworzyły z przesunięciem

Reply:
słuszna uwaga, jednak odłożył bym to na poczet nowego ticketu, gdyż należy wziąć pod uwagę które notatki zostaną przesunięte (te z góry, czy środka)